packages/caip/package.json

```diff
@@ -22,10 +22,14 @@
     "dev": "tsc --watch",
     "generate-coingecko": "yarn build && node dist/adapters/coingecko/generate.js",
     "generate-coincap": "yarn build && node dist/adapters/coincap/generate.js",
+    "generate-yearn": "yarn build && node dist/adapters/yearn/generate.js",
+    "generate": "yarn build && node dist/adapters/coingecko/generate.js && node dist/adatpters/coincap/generate.js && node dist/adapters/yearn/generate.js",
     "test": "jest test",
     "type-check": "tsc --project ./tsconfig.json --noEmit"
   },
-  "dependencies": {},
+  "dependencies": {
```

Comment:
actually, can we make this a dev dep please? it's only needed for generate script which is most def a dev task

Reply:
@0xdef1cafe Good call, done.